projects/packages/my-jetpack/_inc/components/product-interstitial/config.ts

```diff
@@ -0,0 +1,187 @@
+import { ComponentType } from 'react';
+import SearchLogo from './logos/search-logo';
+import SocialLogo from './logos/social-logo';
+
+/**
+ * Configuration types for product pricing tables
+ */
+export interface FeatureTier {
+	included: boolean;
+	label: string;
+}
+
+export interface ProductFeature {
+	name: string;
+	tooltipInfo?: string;
+	free: FeatureTier;
+	paid: FeatureTier;
+	bundle: FeatureTier;
+}
+
+export interface ProductTier {
+	name: string;
+	cta: string;
+}
+
+export interface ProductConfig {
+	title: string;
+	logo: ComponentType< { height?: number } >;
+	bundle: string;
+	features: ProductFeature[];
+	tiers: {
+		free: ProductTier;
+		paid: ProductTier;
+		bundle: ProductTier;
+	};
+}
+
+export interface ProductConfigs {
+	[ productSlug: string ]: ProductConfig;
+}
+
+// Product configuration for pricing tables
+export const PRODUCT_CONFIGS: ProductConfigs = {
+	social: {
+		title: 'Publish once. Share everywhere.',
```

Comment:
We will need to translate these strings. Therefore, it's better to return these in a function instead of a constant, so that it's translated when the translations are loaded async.